### DISCUSSION TITLE
Request for Assistance with GitHub Education Community Access

### DISCUSSION THREAD

**Discussion by Antbasso** [community/community#144912] [GitHub Education]
### Hi everyone,

I just wanted to take a moment to thank you for the warm welcome to the GitHub Education community. I'm really excited to be a part of this community and share my ideas and learn from you.

However, I'm facing a bit of an issue. It's been three days since my student profile was validated, but I still don't have access to the various features offered. I understand that it may take some time, but I'm a bit pressed to get it today as my teacher has assigned an activity this afternoon and I'll need these features to participate.

Could you please help me resolve this issue? I'd be really grateful for your assistance and I'm happy to provide any necessary information to resolve this problem.

Thank you in advance for your understanding and help.

Best regards, Basso Antoine

**Comment by the-bipu** [comment]
are you doing badge farming as i can't see this one as your primary account? 😒

**Reply by Antbasso** [reply]
It's just an account I made at the start of the year for studying.

**Comment by davevad93** [comment]
Hi @Antbasso ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Antbasso was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Antbasso** [reply]
It's a pity it's so long, thank you for your reply.

**Reply by davevad93** [reply]
You're welcome @Antbasso , don't worry you'll soon have your benefits! 👍

**Comment by smithappdev** [comment]
Hi @Antbasso and @davevad93!

Thank you so much for the helpful insights and references! I appreciate you pointing out the status label update and sharing those discussion threads and FAQs. It's good to know that waiting times can vary and that there are resources to refer to while waiting.

@Antbasso, I’d echo @davevad93 's advice about double-checking your profile details to make sure everything is correctly filled in—it could help avoid further delays. Hopefully, your benefits will activate soon so you can participate in your activity without any trouble.

Best of luck, and let us know how things go!
